src/background.js

```diff
@@ -196,3 +199,20 @@ browser.storage.local.get()
 browser.browserAction.onClicked.addListener(() => {
     browser.sidebarAction.open();
 });
+
+// context menu for 'Send to Notes'
+browser.contextMenus.create({
+  id: 'send-to-notes',
+  title: 'Send to Notes',
```

Comment:
Good label!

Reply:
Might be nice to integrate i18n to translate `title: 'Send to Notes'` before merging.